scripts/enrich_event_feed.py

```
@@ -0,0 +1,414 @@
+
+    def __init__(self) -> None:
+        super().__init__(convert_charrefs=True)
+        self.page = ParsedPage()
+        self._root_tag: Optional[str] = None
+        self._depth = 0
+        self._parts: List[str] = []
+        self._section = "intro"
+        self._skip_depth = 0
+        self._seen_h1 = False
+
+    def handle_starttag(self, tag: str, attrs) -> None:
```

Ruff (0.16.1)

[warning] Unused method argument: `attrs`

(ARG002)